lib/resultsCapturer.js

```diff
@@ -235,6 +235,14 @@ function commonStepFinished({
       break;
 
     case Cucumber.Status.AMBIGUOUS:
+      emitterEvent = 'testFail';
+      scenarioFailed = true;
+      stepResults.assertions.push({
+      passed: false,
+      errorMsg: exception ? exception.message : 'Ambiguous step found.',
```

Comment:
I think `exception?.message || 'Ambigous step found` will be better. It also handles the case where message is undefined or empty.

Reply:
I agree!